config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
Looks like github hid a question. `__param_` is the first parameter (not the last as julius initially suggested) so that if we ever want to extend this functionality the next parameter will be `__param2_` which is reasonable for an advanced feature that it's not clear we'll ever need.